_posts/2019-08-06-serverless-static-sites.md

```diff
@@ -0,0 +1,266 @@
+```sh
+kubectl get svc -n default
+NAME                                         TYPE           CLUSTER-IP       EXTERNAL-IP      PORT(S)                      AGE
+nginxingress-nginx-ingress-controller        LoadBalancer   10.245.116.147   159.89.221.240   80:31113/TCP,443:30596/TCP   2d21h
+```
+
+You can see that my external IP address is: `159.89.221.240`.  
+If you are getting started with DNS management I recommend [DigitalOcean](https://digitalocean.com) which is free and easy to use. Here is how I use the CLI to create the record for `my-site.matiaspan.dev`:
+```sh
+doctl compute domain create my-site.matiaspan.dev --ip-address 159.89.221.240
+
+Domain                     TTL
+my-site.matiaspan.dev      0
+```
+And this is how the UI looks like:
+
+![DigitalOcean control panel for DNS management](/images/2019-serverless-static-sites/digital-ocean-dns-management-panel.png)
+
+## Map the custom domain with a FunctionIngress
+Now it's time to map the custom domain with a `FunctionIngress`, just like Alex showed in the [previous blog post](https://www.openfaas.com/blog/custom-domains-function-ingress/). 
+
+Save the following in a YAML file `example-site-fni.yml`:
+```yaml
+apiVersion: openfaas.com/v1alpha2
+kind: FunctionIngress
+metadata:
+  name: example-site-tls
+  namespace: openfaas
+spec:
+  domain: "my-site.matiaspan.dev"
+  function: "example-site"
+  ingressType: "nginx"
+  tls:
+    enabled: true
+    issuerRef:
+      name: "letsencrypt-prod"
+      kind: "Issuer"
+```
+
+* For the `name` I used a convention of the function’s name plus a suffix of -tls if using TLS.
+* Edit the `domain` to point as your own DNS A record or sub-domain
+* For the `issuerRef`, you can use the `-staging` or `-prod` issuer which you set up earlier using the OpenFaaS docs. If you are not using TLS then remove the `tls` section from the file.
+
+Now apply the file with `kubectl apply -f example-site-fni.yml`
+
+## Check out your brand new site!
+After creating the `FunctionIngress` our IngressOperator will detect the recently created CRD and create an ingress record. If you are using TLS this ingress will be decorated with annotations that the CertManager then detects and creates the TLS certificate for your site.
+
+Check the ingress record:
+```sh
+kubectl get ingress -n openfaas
+NAME                  HOSTS                   ADDRESS          PORTS     AGE
+example-site-tls      my-site.matiaspan.dev   165.22.164.208   80, 443   56s
+```
+Now the certificate:
+```sh
+kubectl get cert -n openfaas
+NAME                    READY   SECRET                  AGE
+example-site-tls-cert   True    example-site-tls-cert   89s
+```
+If you modify or delete the `FunctionIngress` then the certificate and the ingress will also be affected.
+
+Navigate to your domain and check out your new site!
+
+![Example website deployed at my-site.matiaspan.dev](/images/2019-serverless-static-sites/hugo-example-site.png)
```

Comment:
![image](https://user-images.githubusercontent.com/34942004/62835573-d6519b00-bc62-11e9-9ed7-2538a7e61922.png)
I believe this is broken

Reply:
@matipan please can you take a look at this feedback? ^